<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJack</class>
 <widget class="QWidget" name="BlackJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="BackToMain">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>61</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background: transparent;
	background-image: url(:/resources/png/undo.png);
	border-style: solid;
    border-width: 0px;
	border-color: #051a39;
}

QPushButton:hover {
	background: transparent;
	background-image: url(:/resources/png/undo_color.png);
	border-style: solid;
    border-width: 0px;
	border-color: #051a39;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>536</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>178</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="Result_label">
       <property name="minimumSize">
        <size>
         <width>291</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>291</width>
         <height>121</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>258</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>7</width>
       <height>601</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>310</width>
         <height>260</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>310</width>
         <height>260</height>
        </size>
       </property>
       <widget class="QLabel" name="Dealer_Card_Label5">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Dealer_Card_Label4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Dealer_Card_Label2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Dealer_Card_Label3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Dealer_Card_Label1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>100</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>Dealer_Card_Label1</zorder>
       <zorder>Dealer_Card_Label2</zorder>
       <zorder>Dealer_Card_Label3</zorder>
       <zorder>Dealer_Card_Label4</zorder>
       <zorder>Dealer_Card_Label5</zorder>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="dealer_score_label">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>310</width>
         <height>260</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>310</width>
         <height>260</height>
        </size>
       </property>
       <widget class="QLabel" name="Fiveth_Card_Label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>150</width>
          <height>225</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="First_Card_Label">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>20</y>
          <width>150</width>
          <height>225</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Third_Card_Label">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>20</y>
          <width>150</width>
          <height>225</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Second_Card_Label">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>20</y>
          <width>150</width>
          <height>225</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="Fourth_Card_Label">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>20</y>
          <width>150</width>
          <height>225</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>First_Card_Label</zorder>
       <zorder>Second_Card_Label</zorder>
       <zorder>Third_Card_Label</zorder>
       <zorder>Fourth_Card_Label</zorder>
       <zorder>Fiveth_Card_Label</zorder>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="player_score_label">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>7</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>182</width>
         <height>601</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="info">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background: transparent;
	background-image: url(:/resources/png/info.png);
	border-style: solid;
    border-width: 0px;
	border-color: #051a39;
}

QPushButton:hover {
	background: transparent;
	background-image: url(:/resources/png/information.png);
	border-style: solid;
    border-width: 0px;
	border-color: #051a39;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>548</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>980</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #92BA92;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>178</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="summa_stavky">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 15px;
background :#C5c5c5;
color:black;
background :#E3FCBF;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string>Сума ставки</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Stay">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:5px;
background-color:#C5c5c5;
color:black;
background :#E3FCBF;</string>
          </property>
          <property name="text">
           <string>Досить!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StartGame">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:25px;
background-color:#C5c5c5;
color:black;
background :#E3FCBF;</string>
          </property>
          <property name="text">
           <string>Грати</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:5px;
background-color:#C5c5c5;
color:black;
background :#E3FCBF;</string>
          </property>
          <property name="text">
           <string>Добавити</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="userbalance">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #C5c5c5;
font: 16pt &quot;Tahoma&quot;;
border-radius: 15px;
color:black;
background :#E3FCBF;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
